<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InsertDataSample</class>
 <widget class="QDialog" name="InsertDataSample">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>246</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>451</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>数据处理</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>采集最小值</string>
      </property>
      <property name="buddy">
       <cstring>inputMin</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="inputMin">
      <property name="inputMethodHints">
       <set>Qt::ImhPreferNumbers</set>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>工程最小值</string>
      </property>
      <property name="buddy">
       <cstring>outputMin</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLineEdit" name="outputMin">
      <property name="inputMethodHints">
       <set>Qt::ImhPreferNumbers</set>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>采集最大值</string>
      </property>
      <property name="buddy">
       <cstring>inputMax</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="inputMax">
      <property name="inputMethodHints">
       <set>Qt::ImhPreferNumbers</set>
      </property>
      <property name="text">
       <string>100</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>工程最大值</string>
      </property>
      <property name="buddy">
       <cstring>outputMax</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLineEdit" name="outputMax">
      <property name="inputMethodHints">
       <set>Qt::ImhPreferNumbers</set>
      </property>
      <property name="text">
       <string>100</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>210</y>
     <width>158</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btn_ok">
      <property name="text">
       <string>确定</string>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_cancel">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>数据类型 </string>
   </property>
   <property name="buddy">
    <cstring>combo_datatype</cstring>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_addr">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../resourse.qrc">
     <normaloff>:/addressintput/images/address.bmp</normaloff>:/addressintput/images/address.bmp</iconset>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_datatype">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>20</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>16位整数</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>32位整数</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>16位正整数</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>32位正整数</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>16位BCD码</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>32位BCD码</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>32位浮点数</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ASCII字符串</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>UNICODE字符串</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="edit_name">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_addr">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>LW0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>85</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>名 称</string>
   </property>
   <property name="buddy">
    <cstring>spin_length</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>21</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>地 址</string>
   </property>
   <property name="buddy">
    <cstring>edit_addr</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>53</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>个 数</string>
   </property>
   <property name="buddy">
    <cstring>spin_length</cstring>
   </property>
  </widget>
  <widget class="QSpinBox" name="spin_length">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>256</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spin_decLen">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>50</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>4</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>80</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>四舍五入</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>50</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>小数位数</string>
   </property>
   <property name="buddy">
    <cstring>spin_length</cstring>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>edit_addr</tabstop>
  <tabstop>btn_addr</tabstop>
  <tabstop>combo_datatype</tabstop>
  <tabstop>spin_length</tabstop>
  <tabstop>btn_ok</tabstop>
  <tabstop>btn_cancel</tabstop>
 </tabstops>
 <resources>
  <include location="../../resourse.qrc"/>
 </resources>
 <connections/>
</ui>
